pipelines/controller/config/application.yaml

```diff
@@ -40,7 +40,14 @@ fhirdata:
   # For *nix file systems, dwhRootPrefix must be of the format
   # "/<baseDirPath>/<prefix>" and can be absolute or relative. <baseDirPath>
   # is required for *nix, and must contain 1 or more directory names.
-  # dwhRootPrefix: "gs://fhir-test-analytics/config/controller_DWH_ORIG"
+  # Sample 1 : "/fhir-test-analytics/dwh/controller_DWH_ORIG"
+  # Sample 2 : "dwh/controller_DWH_ORIG"
+  #
+  # For Windows file systems, dwhRootPrefix must be of the format
+  # "<baseDirPath>\\<prefix>" and can be absolute or relative. <baseDirPath>
```

Comment:
Why do we need two `\`s as the separator? I mean isn't `C:\fhir-test-analytics\dwh\controller_DWH_ORIG` valid? (ditto below)

Reply:
The character `\` is treated as a escape character, hence we use two `\`s to override the behaviour.

Reply:
Apparently, using double quotes in yaml will treat any backslash `\` as an escape character. However, using `single-quotes` and `no-quotes` has no effect on the backslash '\' as per the documentation over [here](https://yaml.org/spec/1.2-old/spec.html#id2788097) and [here](https://yaml.org/spec/1.2-old/spec.html#id2788859).

Reply:
Oh, I see, I didn't realize that this applies to double-quote strings in YAML too; thanks for clarifying.